UNLV-CS472-672/2024-S-GROUP5-Munch
# Created test directory and create test cases for post functionality

## Description / Changes Made
- Create unit_tests directory to work with run_tests.py
- Added tests cases for server.py functions including get_post(), create_post(), update_post(), and delete_post()

## How to Test

1. Run backend API server by using python3 server.py -m http.server
2. On a separate terminal run python3 run_tests.py
3. Check if tests pass or fail, then also check coverage report.

## Checklist
